pkg/windows/windows.go

```diff
@@ -698,6 +684,28 @@ func (vm *windows) ensureServiceNotRunning(svc *service) error {
 
 }
 
+// ensureServiceIsRemoved ensures that the given service installed by WMCO is removed from the instance
+func (vm *windows) ensureServiceIsRemoved(svcName string) error {
+	svc := &service{name: svcName}
+	// If the service is not installed, do nothing
+	exists, err := vm.serviceExists(svcName)
+	if err != nil {
+		return errors.Wrapf(err, "error checking if %s Windows service exists", svc.name)
+	}
+	if !exists {
+		return nil
+	}
+	// Make sure the service is stopped before we attempt to delete it
+	if err := vm.ensureServiceNotRunning(svc); err != nil {
+		return errors.Wrapf(err, "error stopping %s Windows service", svc.name)
+	}
+	if err := vm.deleteService(svc); err != nil {
+		return errors.Wrapf(err, "error deleting %s Windows service", svc.name)
+	}
+	vm.log.Info("deconfigured", "service", svc.name)
+	return nil
+}
```

Comment:
nit: This is not new code, but because it was moved within the file it is presenting as if it is, making the PR longer to review.